alpine/Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM alpine:3.3
 
+RUN apk --update add \
+	ca-certificates \
+	&& rm -rf /var/cache/apk/*
```

Comment:
good call!

On Wed, Apr 13, 2016 at 4:43 PM, Tianon Gravi <notifications@github.com>
wrote:

> In alpine/Dockerfile
> <https://github.com/jfrazelle/irssi/pull/8#discussion_r59644954>:
>
> > @@ -1,5 +1,9 @@
> >  FROM alpine:3.3
> >
> > +RUN apk --update add \
> > +	ca-certificates \
> > +	&& rm -rf /var/cache/apk/*
>
> You should use apk add --no-cache ca-certificates here and then we don't
> need either --update or the rm [image: :+1:]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jfrazelle/irssi/pull/8/files/669c2b52e5db58804eb373901b79c8e49826eb27#r59644954>
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>